README.md

```diff
@@ -29,7 +29,25 @@ Supports utility projects that do not compile an assembly.
 
 Supports staging artifacts from build outputs.
 
-## What Are MSBuild SDKS?
+## How can I use these SDKs?
+
+Start your project file with:
+
+```xml
+<Project Sdk="<package-name>/<version>">
```

Comment:
I generally recommend people use `global.json` to specify the versions so that its only defined in one place.  That matters much more for Traversal and NoTargets and much less for CentralPackageVersions.  Should this explain the recommended way more to emphasize the use of global.json instead of the version in the project?  Or do you think that makes it harder to understand?